src/legacy/core_plugins/kibana/public/local_application_service/local_application_service.ts

```diff
@@ -106,7 +106,9 @@ export class LocalApplicationService {
         template: '<span></span>',
         controller($location: ILocationService) {
           const newPath = forwardDefinition.rewritePath($location.url());
-          npStart.core.application.navigateToApp(forwardDefinition.newAppId, { path: newPath });
+          window.location.replace(
+            npStart.core.http.basePath.prepend(`/app/${forwardDefinition.newAppId}/${newPath}`)
+          );
```

Comment:
Could you give me some context on this change to `location.replace`? What is still using this now? Just management?

Reply:
I think I changed the implementation here because of the bug in `navigateToApp` when switching to legacy applications (the one you fixed already). I will check whether it's still necessary.